### PR TITLE
[12.0][FIX] web_widget_one2many_product_picker: Dulicity records on mobile

### PR DESCRIPTION
**Pull request by Tardo**
In mobile the deactivation of the buttons is not working properly, so now we ensure that only the last action is triggered using debounce.

cc @tecnativa TT28259

### CODE REVIEW

**Review by pedrobaeza — APPROVED** [review]

File: web_widget_one2many_product_picker/static/src/js/views/One2ManyProductPicker/quick_create_form_view.js [hunk 1]
```diff
@@ -301,10 +301,12 @@ odoo.define("web_widget_one2many_product_picker.ProductPickerQuickCreateFormView
                         success_callback: function () {
                             self.trigger_up("update_quick_record", {
                                 id: record.id,
+                                callback: function () {
+                                    self.model.unsetDirty(self.handle);
+                                    //self._updateButtons();
```
Comment (chienandalu) [inline web_widget_one2many_product_picker/static/src/js/views/One2ManyProductPicker/quick_create_form_view.js]: Should it be uncommented or go away? :thinking:

File: web_widget_one2many_product_picker/static/src/js/views/One2ManyProductPicker/quick_create_form_view.js [hunk 2]
```diff
@@ -259,10 +259,12 @@ odoo.define("web_widget_one2many_product_picker.ProductPickerQuickCreateFormView
                             success_callback: function () {
                                 self.trigger_up("create_quick_record", {
                                     id: record.id,
+                                    callback: function () {
+                                        self.model.unsetDirty(self.handle);
+                                        //self._updateButtons();
```
Comment (chienandalu) [inline web_widget_one2many_product_picker/static/src/js/views/One2ManyProductPicker/quick_create_form_view.js]: Same here

**Review by chienandalu — APPROVED** [review]
Tested. Check my two comments anyway

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**